packages/field-base/src/field-mixin.js

```diff
@@ -266,6 +268,22 @@ export const FieldMixin = (superclass) =>
       );
     }
 
+    /**
+     * @protected
+     * @override
+     */
+    _toggleHasLabelAttribute() {
+      super._toggleHasLabelAttribute();
+
+      // Label ID should be only added when the label content is present.
+      // Otherwise, it may conflict with an `aria-label` attribute possibly added by the user.
```

Comment:
I'm a bit lost here: the comment says about `aria-label` but we don't check for it. Should this be updated?

Reply:
Good point, I'll fix the comment. On a related note, should we follow the same principle for the error id and add it only when the `has-error-message` attribute is present? I'm not sure if it makes sense in the case when the error message is not provided.

Reply:
> should we follow the same principle for the error id and add it only when the has-error-message attribute is present?

Yes, I'd say we should. We can change that in a separate PR though.